store/src/java/com/zimbra/qa/unittest/TestJaxb.java

```diff
@@ -128,7 +129,12 @@ public void tearDown() throws Exception {
     }
 
     private static String getCN(ZMailbox mbox) throws ServiceException {
-        String cn = mbox.getAccountInfo(false).getAttrs().get("cn").get(0);
+        String cn = null;
+        try {
+            cn = mbox.getAccountInfo(false).getAttrs().get("cn").get(0);
+        } catch (ExecutionException e) {
```

Comment:
As suggested elsewhere, should use something that handles ExecutionException at a higher level.